signingscript/docker.d/passwords.yml

```diff
@@ -95,12 +95,19 @@ in:
              {"$eval": "AUTOGRAPH_REFERENCE_BROWSER_PASSWORD"},
              ["autograph_apk"]
             ]
+        # TODO Bug 1791397: Remove the following mapping
         project:mobile:android-components:releng:signing:cert:dep-signing:
           - ["https://autograph-external.prod.autograph.services.mozaws.net",
              {"$eval": "AUTOGRAPH_GPG_USERNAME"},
              {"$eval": "AUTOGRAPH_GPG_PASSWORD"},
              ["autograph_gpg"]
             ]
+        project:mobile:firefox-android:releng:signing:cert:dep-signing:
```

Comment:
We'll want new cot restricted trees+scopes when you're ready to move to prod signing.

Reply:
Good call! Done in https://github.com/mozilla-releng/scriptworker/pull/561